.github/workflows/ci.yml

```diff
@@ -32,17 +27,15 @@ jobs:
           cargo check --all-features
       - name: test
         run: |
-          cargo install cargo-binutils@0.3.6
-          cargo test 
+          # TODO: use a published version, or at least a merged branch
+          cargo install --git https://github.com/jyn514/cargo-binutils/ --branch metadata
```

Comment:
Just wondering why linux needs this version but macos and windows are fine with 0.3.6?